src/ngx_http_lua_socket_tcp.c

```diff
@@ -1002,6 +1004,49 @@ ngx_http_lua_socket_resolve_handler(ngx_resolver_ctx_t *ctx)
 }
 
 
+static void
+ngx_http_lua_socket_init_peer_connection_addr_text(ngx_peer_connection_t *pc)
+{
+    ngx_connection_t            *c;
+    size_t                       addr_text_max_len;
+
+    c = pc->connection;
+
+    switch (pc->sockaddr->sa_family) {
+
+#if (NGX_HAVE_INET6)
+    case AF_INET6:
+        addr_text_max_len = NGX_INET6_ADDRSTRLEN;
+        break;
+#endif
+#if (NGX_HAVE_UNIX_DOMAIN)
+
+    case AF_UNIX:
+        addr_text_max_len = NGX_UNIX_ADDRSTRLEN;
+        break;
+#endif
+    case AF_INET:
```

Comment:
Style: need a blank line before this `case` statement as well. I'll fix it locally myself.